dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/util/ContentletUtilTest.java

```diff
@@ -147,14 +146,19 @@ public void validateContentPrintableMapMethodReturnProperCategories(
                 verifyCategoriesForField(contentPrintableMap, CATEGORY_NAME_CONTENT_BELTS,
                         homeCategory, flightsCategory);
             } else {
-                assertEquals( popularCategory.getCategoryName(), contentPrintableMap.get( CATEGORY_NAME_CONTENT ) );
-                final String [] resultCategories = contentPrintableMap.get(CATEGORY_NAME_CONTENT_BELTS)
-                        .toString().split(",\\s?");
-                assertEquals(2, resultCategories.length);
-                assertTrue(Arrays.stream(resultCategories)
-                        .anyMatch(c -> c.equals(homeCategory.getCategoryName())));
-                assertTrue(Arrays.stream(resultCategories)
-                        .anyMatch(c -> c.equals(flightsCategory.getCategoryName())));
+                List<Map<String, Object>> list = (List) contentPrintableMap
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'list' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)